silverstripe/silverstripe-admin
# FIX disable options not working issue on TreeDropdownField

## Description
This PR is for fixing an issue related to TreeDropdownField which the disabled options are treated as enabled in the cms field. 

## Manual testing steps

- Setup a SilverStripe site for v5.3
- Checkout the branch for the module
- Create a test TreeDropdownField for a test model.
- Use `TreeDropdownField::setDisableFunction()` to disable some options and check whether they are disabled in CMS.

## Issues

- https://github.com/silverstripe/silverstripe-framework/issues/11208
